src/providers/mod.rs

```diff
@@ -51,6 +52,7 @@ use crate::network;
 const ENV_PREFIX: &str = "AFTERBURN_";
 #[cfg(feature = "cl-legacy")]
 const ENV_PREFIX: &str = "COREOS_";
+const AFTERBURN_SSH_ATHORIZED_KEYS_MESSAGEID: &str = "0f7d7a502f2d433caa1323440a6b4190";
```

Comment:
Typo: `AUTHORIZED`.

Also, please add a docstring to record what's the purpose of this.